### ISSUE TITLE
Missing interface for purge_by_key - for purge only tokens

### ISSUE THREAD

**Issue by ranguard** [fastly/fastly-perl#27]
Creating a token that is limited to a service and [purge scope](https://docs.fastly.com/api/auth#scope) means that whilst you can do:
```perl
    $fastly->purge('http://www.example.com');
```
You can't read the service:
```perl
   my $service = $fastly->get_service($service_id);
```
So you can't call:
```perl
    $service->purge_by_key('article-1');
```

It's easy enough to do using an API call to `/service/${service_id}/purge/${key}`, but I can't see a nice way of doing it through this module.

**Comment by ranguard** [comment]
https://metacpan.org/pod/MooseX::Fastly::Role now splits out and does purging through HTTP::Tiny and access to Net::Fastly seperatly.

**Comment by Integralist** [comment]
Closing this issue now we've published 2.01.
